lucene/core/src/java/org/apache/lucene/index/ExitableDirectoryReader.java

```diff
@@ -100,13 +109,156 @@ public CacheHelper getCoreCacheHelper() {
 
   }
 
+  /**
+   * Wrapper class for another PointValues implementation that is used by ExitableFields.
+   */
+  public static class ExitablePointValues extends PointValues {
+
+    private final PointValues in;
+    private final QueryTimeout queryTimeout;
+
+    public ExitablePointValues(PointValues in, QueryTimeout queryTimeout) {
```

Comment:
I'm getting precommit failures about missing javadocs for this constructor, aren't you?

Reply:
Issue found: I forgot to add a `set -e` statement in my tiny bash script below to stop on first failure.

Let me run another `precommit` Ant task and push.

```bash
#!/usr/bin/env bash

set -e

cd ${HOME}/git/lucene-solr
find . -name "*.lck" -delete
ant clean compile precommit
ant -f lucene/build.xml test
```

Reply:
That's it, documentation fixed and `precommit` task really green :sweat_smile: thanks!

Reply:
But isn't it worse now? precommit is satisfied, but nothing more than (obvious) noise was added.

Reply:
I'm not sure to understand, the `set -e` flag prevents the second Ant command line from being run if the `precommit` task fails, and therefore the latest output in my terminal is a build failure.

The `set -x` (debug option) flag would add a lot noise.